internal/store/postgres/servicedata.go

```diff
@@ -24,6 +24,30 @@ type ServiceData struct {
 	Value string `db:"value"`
 }
 
+type GetServiceData struct {
```

Comment:
This should be the map of Service Data to the table itself and not be a method model. 

```suggestion
type ServiceData struct {
```